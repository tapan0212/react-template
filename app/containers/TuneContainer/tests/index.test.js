/**
 *
 * Tests for TuneContainer
 *
 *
 */

import React from 'react';
import { renderProvider } from '@utils/testUtils';
// import { fireEvent } from '@testing-library/dom'
import { TuneContainerTest as TuneContainer } from '../index';

describe('<TuneContainer /> container tests', () => {
  // let submitSpy
  const mockSongData = {
    trackId: 1,
    artistName: 'Ed',
    trackName: 'Perfect',
    releaseDate: '2017-05-10'
  };
  beforeEach(() => {
    // submitSpy = jest.fn()
  });
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<TuneContainer selectedTune={mockSongData} />);
    expect(baseElement).toMatchSnapshot();
  });
});
