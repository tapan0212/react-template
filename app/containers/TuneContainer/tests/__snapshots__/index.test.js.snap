// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TuneContainer /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div>
      <div
        class="TuneContainer__CustomInfoCard-sc-1u7g34z-0 sPugY"
      >
        <div
          class="TuneContainer__CustomCard-sc-1u7g34z-1 bcKNLx"
        >
          <div
            class="TuneContainer__CustomImg-sc-1u7g34z-2 cfMkcz"
          >
            <img
              alt=""
            />
          </div>
          <div>
            <p
              class="T__StyledText-gjlic1-0 cbAvsY"
              data-testid="t"
            >
              Tune: Perfect
            </p>
          </div>
          <div>
            <p
              class="T__StyledText-gjlic1-0 cbAvsY"
              data-testid="t"
            >
              Artist Name: Ed
            </p>
          </div>
          <div>
            <p
              class="T__StyledText-gjlic1-0 cbAvsY"
              data-testid="t"
            >
              Release Year: 2017
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
